### ISSUE TITLE
Is there a way to get the hinge points as a list?

### ISSUE THREAD

**Issue by apatange-source** [scikit-learn-contrib/py-earth#219]

**Comment by apatange-source** [comment]
I need a way to visualize the splines and the hinge points

**Comment by kevin-dietz** [comment]
@apatange-source 
The easiest way I know how is to look at the basis, which contains all of the model terms.  You would want to filter out the pruned terms, as they are not actually used in the final mdoel

```
model = Earth()
model.fit(x_train, y_train)
model.named_steps['earth'].basis_ 
```

Alternatively, you can look at the model equation and do some regex to get what you need:
```print(str(export.export_sympy(model)))```

**Comment by apatange-source** [comment]
@kevin-dietz I wanted the individual x-axis points. 

I got them by using `model.basis_[i].get_knot()` and `model.basis_[i].get_variable()`

Thank you anyway :D
